Import squad jobs

jscook2345 opened the pull request:
Moving scripts from gitlab to here: https://gitlab.com/Linaro/lkft/reports/squad-jobs

liuyq commented:
@jscook2345 where can I find the change on the README.md? not sure if there is an example on how to run the tools

Commit:
all: import scripts from squad-jobs

Signed-off-by: Justin Cook <justin.cook@linaro.org>

jscook2345 commented:
> @jscook2345 where can I find the change on the README.md? not sure if there is an example on how to run the tools

`README.md` is now added and updated. Thanks!

chaws commented:
It's funny how each of those scripts would fit nicely in squad-client/commands :p.
LGTM!

If you find that any of these scripts should live in squad-client, we can arrange that!

jscook2345 commented:
@chaws 

There's a few advantages here, but I definitely agree.

The idea is to create a playground of scripts and see which ones get a lot of use. Those we can promote to our toolset, squad-client, squad-report, or squad itself.

We also get to prototype and change things around a bit more when they are in script form until we can get a stable api.
